### ISSUE TITLE
Fatal error PHPUnit_Framework_TestResult

### ISSUE THREAD

**Issue by eugene-manuilov** [ngraf/slack-codeception-extension#7]
Hey, 

I have gotten following error when I try to use your extension with Codeception ^2.4. Looks like your extension expects to receive an older version of PHPUnit results class.

```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Codeception\Extension\SlackExtension::sendSuccessMessage() must be an instance of PHPUnit_Framework_TestResult, instance of PHPUnit\Framework\TestResult given, called in .../vendor/ngraf/slack-codeception-extension/src/Extension/SlackExtension.php on line 206 and defined in .../vendor/ngraf/slack-codeception-extension/src/Extension/SlackExtension.php:226
Stack trace:
#0 .../vendor/ngraf/slack-codeception-extension/src/Extension/SlackExtension.php(206): Codeception\Extension\SlackExtension->sendSuccessMessage(Object(PHPUnit\Framework\TestResult))
#1 .../vendor/symfony/event-dispatcher/EventDispatcher.php(212): Codeception\Extension\SlackExtension->sendTestResults(Object(Codeception\Event\PrintResultEvent), 'result.print.af...', Object(Symfony\Component\EventDispatcher\EventDispatcher))
#2 .../vendor/symfony/event-dispatche in .../vendor/ngraf/slack-codeception-extension/src/Extension/SlackExtension.php on line 226
```

**Comment by ngraf** [comment]
Hi, I am quite sure I fixed this 3 days ago in version 1.4.3 of "ngraf/slack-codeception-extension".
https://github.com/ngraf/slack-codeception-extension/releases/tag/1.4.3

Please run composer update and report back.

**Comment by ngraf** [comment]
Closed. Fixed in latest version.
